test/tests/functional/pbs_qmgr.py

```diff
@@ -124,3 +125,30 @@ def test_listcmd_whitespaces(self):
             if vndef_file is not None:
                 self.mom.delete_vnodes()
                 self.server.manager(MGR_CMD_DELETE, VNODE, id=nodename)
+
+    def test_multi_attributes(self):
+        """
+        Test to verify that if multiple attributes are set
+        simultaneously and out of which one fail then none
+        will be set.
+        """
+
+        a = {'queue_type': 'e', 'enabled': 't', 'started': 't'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='workq2')
+
+        a = {'partition': 'foo'}
+        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+
+        a = {'partition': 'bar'}
+        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq2')
+
+        a = {'queue': 'workq', 'partition': 'bar'}
+        try:
+            self.server.manager(MGR_CMD_SET, NODE, a,
+                                id=self.mom.shortname)
+        except PbsManagerError as e:
+            msg = "Partition (null) is not part of "
```

Comment:
'(null)' is a bug.  You shouldn't depend on it for this test.  This test will fail once it is fixed.

Reply:
How do you want me to handle this? one option is to not look for the error message and second option is while PP-1073 is fixed update this test as well.

Reply:
How about just searching for the partial string?  Drop the "Partition (null)" part.  You can put in a comment in the test that you are searching for a partial string due to PP-1073.  Add a comment in PP-1073 that this test needs to be updated when it is fixed.